packages/experiments-realm/Vendor/1.json

```diff
@@ -78,17 +82,20 @@
             "bic": null
           }
         }
-      ]
+      ],
+      "title": null,
+      "description": null,
+      "thumbnailURL": null
     },
     "relationships": {
       "preferredPaymentMethod.cryptoPayment.chain": {
         "links": {
-          "self": "../Chain/1"
+          "self": "http://localhost:4201/experiments/Chain/1"
         }
       },
       "preferredPaymentMethod.cryptoPayment.token": {
         "links": {
-          "self": "../Token/1"
+          "self": "http://localhost:4201/experiments/Token/1"
```

Comment:
Will these affect the relationships in staging etc?

Reply:
Good catch. Yes it will